### PR TITLE
OADP#2671: Added new assembly about Kopia

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.11+

Issue: [OADP-2671](https://issues.redhat.com//browse/OADP-2671)

Link to docs preview: https://68304--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-about-kopia

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by stillalearner — APPROVED** [review]
lgtm

**Review by weshayutin — APPROVED** [review]
LGTM, thank you!

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-about-kopia.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+[id="oadp-about-kopia"]
+= About Kopia
+include::_attributes/common-attributes.adoc[]
+:context: oadp-about-kopia
+
+toc::[]
+
+Kopia is a fast and secure open-source backup and restore tool that allows you to create encrypted snapshots of your data and save the snapshots to remote or cloud storage of your choice.
+
+Kopia supports network and local storage locations, as well as many cloud or remote storage locations, including:
```
Comment (mburke5678) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-about-kopia.adoc:11]: Per ISG, do not use _as well as_ to mean _and_.
```suggestion
Kopia supports network and local storage locations, and many cloud or remote storage locations, including:
```

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-about-kopia.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+* Amazon S3 and any cloud storage that is compatible with S3
+* Azure Blob Storage
+* Google Cloud Storage Platform
+
+Kopia uses content-addressable storage for snapshots:
+
+* Each snapshot is always incremental. This means that all data is uploaded once to the repository based on file content, and a file is only uploaded to the repository again if it is modified.
```
Comment (mburke5678) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-about-kopia.adoc:19]:
```suggestion
* Each snapshot is always incremental. This means that all data is uploaded once to the repository, based on file content. A file is only uploaded to the repository again if it is modified. 
```

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-about-kopia.adoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: dpa-sample
+spec:
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+#...
```
Comment (mburke5678) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-about-kopia.adoc:41]: Per [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#signs-and-symbols-in-code-blocks):
```suggestion
# ...
```

**Pull request merged**